README.md

```
@@ -23,10 +23,10 @@ JAM Prize requirements
     - [x] Block import
     - [x] W3F test vectors
     - [x] JAM Conformance Fuzzer
-    - [ ] Performance optimisations
+    - [x] Performance optimisations
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)